README.rst

```diff
@@ -26,6 +26,17 @@ They are available under a `BSD license`_.
 .. contents::
 
 
+Quick Start
+===========
+To profile a python script:
+
+* Install line_profiler: ``pip install line_profiler``.
+
+* Decorate function(s) you want to profile with ``@profile``.
+  This decorater will be made magically available on run.
```

Comment:
I would add parens after magical and give a brief explanation that it adds it as a builtin.

Otherwise it LGTM.

Reply:
how about "LineProfiler will automatically make this decorator available on run." I don't think too much explanation is warranted in a Quick Start, just enough that the person isn't wondering "Do I need to do something like `from line_profiler import profile`?"

Reply:
Sounds good. I want to avoid the term "magic", because (1) it's not and (2) it doesn't give the user an opportunity to learn more. I'd even be happy with keeping magic, but making it a hyperlink that links to where the global injection happens in the code (ideally the code would contain a comment explaining it in detail).

Reply:
Sure, a link would be good; briefly reading through the code, I can't figure out where that happens 😛 . Since code location changes (unless we pin it to a specific commit), it'd be better to provide reference to documentation. However, I don't see this described in any of the documentation, either.

In the sake of being broadly useful without too much effort, let's just settle on some wording without a hyperlink and with an emphasis on being terse. Would we be ok with:

> Decorate function(s) you want to profile with ``@profile``. The decorator will be made automatically available on run.

Reply:
Sounds good to me

Reply:
updated